entrypoint.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
+
+if [[ "${DB_TYPE}" == "postgres" ]]; then
+  echo "Waiting for Postgres..."
+
+  while ! nc -z $DB_HOST $DB_PORT; do
+    sleep 0.1
+  done
+
+  echo "PostgreSQL started"
+fi
+
+python manage.py migrate
+yes | python manage.py create_initial_data
```

Comment:
Do we need to invoke this each time? It seems to me that we can make the `postgres_data` volume persistent.

Reply:
And we can pass ` --noinput` to `migrate`.

Reply:
I can remove the entrypoint script. Could you let me know how to make the postgres_data volume persistent?

Reply:
This looks like an interesting idea but I don't know if it would satisfy what you're looking for: https://stackoverflow.com/a/44283611